.github/workflows/helm-release.yaml

```diff
@@ -0,0 +1,20 @@
+name: helm-release
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Ideally we infer the Loki version. What do you think?

Reply:
What do you mean by that, not sure I'm following?

Reply:
I meant we infer the Loki version by the tag for this flow. This is not really connected to this line though.

Reply:
I don't think we want this workflow to run on tags, we typically push the tag for the release in order to build the binaries and images, and _then_ we update the helm charts/jsonnet in a separate step only after the release artifacts exist.

Reply:
I agree, let's remove the `tags` section alltogehter, which also more closely matches what mimir does:
```yaml  
push:
    branches:
      - main
      - "release-[0-9]+.[0-9]+"
```

Note: we don't wan the `release-` bit either. They've done some work to add releasing the helm chart to their release workflow, but we're not there yet.